src/components/settings/PersonalizedPushSettings.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useContext, useEffect, useMemo, useState } from 'react';
+import { useQuery } from 'react-apollo';
+import { FlatList, StyleSheet } from 'react-native';
+
+import { SettingsContext } from '../../SettingsProvider';
+import { normalize, texts } from '../../config';
+import { getInAppPermission } from '../../pushNotifications';
+import { getQuery, QUERY_TYPES } from '../../queries';
+import { onActivatePushNotifications, onDeactivatePushNotifications } from '../../screens';
+import { LoadingSpinner } from '../LoadingSpinner';
+import { SafeAreaViewFlex } from '../SafeAreaViewFlex';
+import { SettingsToggle } from '../SettingsToggle';
+
+type Category = { id: string; iconName?: string | null; name: string };
+
+export const PersonalizedPushSettings = () => {
+  const { globalSettings } = useContext(SettingsContext);
+  const { settings = {} } = globalSettings;
+  const { personalizedPush = {}, settingsScreenListItemTitles = {} } = settings;
+  const queryVariables = personalizedPush.variables || {
```

Comment:
shouldn't `variables` be part of `categories` object, as it relates to each other? what is the logical explanation to have it next to each other, also regarding future updates?

Reply:
i have changed this in https://github.com/smart-village-solutions/smart-village-app-app/pull/1370 to for example:

```
"personalizedPush": {
  "categories": {
    "queryVariables": {
      "tagList": [
        "news_item"
      ]
    },
    "sectionTitle": "Push-Benachrichtigungen für Nachrichtenkategorien"
  }
}
```